2_run/src/run_glm3.R

```diff
@@ -67,7 +81,7 @@ extract_glm_output <- function(sim_lake_dir, nml_obj, export_fl) {
 #'  the name of the export feather file, its hash (NA if the model run failed), 
 #'  the duration of the model run, whether or not the model run succeeded, 
 #'  and the code returned by the call to GLM3r::run_glm(). 
-run_glm3_model <- function(sim_dir, nml_objs, model_config, export_fl_template) {
+run_glm3_model <- function(sim_dir, nml_objs, model_config, burn_in, burn_out, export_fl_template) {
```

Comment:
Ooo more parameterization 💯

Reply:
Do you want to add defaults here? Or always require a call to `run_glm3_model()` to have them specified? It might be a nice place to retain the 300 & 190 values for future reference.

```r
run_glm3_model <- function(sim_dir, nml_objs, model_config, export_fl_template, burn_in = 300, burn_out = 190)
```